src/sentry/snuba/tasks.py

```diff
@@ -250,14 +252,23 @@ def _create_snql_in_snuba(subscription, snuba_query, snql_query, entity_subscrip
         )
 
     entity_key = get_entity_key_from_request(snql_query)
+
+    if in_random_rollout("snuba.snql.enable-orjson"):
+        post_body = orjson.dumps(body).decode()
+    else:
+        post_body = json.dumps(body)
     response = _snuba_pool.urlopen(
         "POST",
         f"/{snuba_query.dataset}/{entity_key.value}/subscriptions",
-        body=json.dumps(body),
+        body=post_body,
```

Comment:
and it looks like `body` can take bytes directly so the `.decode()` is just wasted cpu